src/jira/client/axios.ts

```diff
@@ -84,16 +100,32 @@ function getErrorMiddleware(logger: Logger) {
 		 * @returns {Promise<Error>} The rejected promise
 		 */
 		(error: AxiosError): Promise<Error> => {
-			if (error?.response) {
-				const status = error.response.status;
+
+			let isWarning = false;
+			let errorMessage = "";
+
+			const status = error?.response?.status;
+
+			if (status) {
 
 				// truncating the detail message returned from Jira to 200 characters
-				const errorMessage = getJiraErrorMessages(status);
-				// Creating an object that isn't of type Error as bunyan handles it differently
-				// Log appropriate level depending on status - WARN: 300-499, ERROR: everything else
-				(status >= 300 && status < 500 ? logger.warn : logger.error)(error, errorMessage);
+				errorMessage = getJiraErrorMessages(status);
```

Comment:
Move this statement out of the `if` block. It will never print the default error message if the status/response is undefined